ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/stats/FilterSelectivityEstimator.java

```diff
@@ -511,7 +511,7 @@ private static double rangedSelectivity(KllFloatsSketch kll, float val1, float v
    * @return the selectivity of a predicate "column &gt; value" in the range [0, 1]
    */
   public static double greaterThanSelectivity(KllFloatsSketch kll, float value) {
-    float max = kll.getMaxValue();
+    float max = kll.getMaxItem();
```

Comment:
https://github.com/apache/datasketches-java/commit/ce9bed7f753ddaa27fc1162f88bae350af08231b#diff-3a7fd0aa04e700d6abbaa5025d434d4098a80d60699b192dc2995a7201ff3aea This commit changed the method name.

Reply:
I didn't do much research, but I think it is ok. 
To be honest, it's hard to explore the reasons for some commits of apache datasketches. Apache datasketches often doesn't give more details about its commit. :(